frontend/src/data/variables/AcsPopulationProvider.ts

```diff
@@ -35,17 +35,49 @@ class AcsPopulationProvider extends VariableProvider {
     super(
       "acs_pop_provider",
       ["population", "population_pct"],
-      ["acs_population-by_race_state_std", "acs_population-by_age_state"]
+      [
+        "acs_population-by_race_state_std",
+        "acs_population-by_race_county_std",
+        "acs_population-by_age_state",
+      ]
     );
   }
 
+  getDatasetId(breakdowns: Breakdowns): string {
+    if (breakdowns.demographicBreakdowns.age.enabled) {
+      return "acs_population-by_age_state";
+    }
+    if (
+      breakdowns.demographicBreakdowns.race_nonstandard.enabled ||
+      breakdowns.demographicBreakdowns.race.enabled
+    ) {
+      return breakdowns.geography === "county"
+        ? "acs_population-by_race_county_std"
+        : "acs_population-by_race_state_std";
+    }
+    return "";
+  }
+
   getDataInternal(
     datasets: Record<string, Dataset>,
     breakdowns: Breakdowns
   ): MetricQueryResponse {
     let df = this.getDataInternalWithoutPercents(datasets, breakdowns);
-    if (breakdowns.filterFips) {
-      df = df.where((row) => row.state_fips === breakdowns.filterFips);
+    const [fipsColumn, geoNameColumn] =
+      breakdowns.geography === "county"
+        ? ["county_fips", "county_name"]
+        : ["state_fips", "state_name"];
+
+    // If requested, filter geography by state or county level
+    if (breakdowns.filterFips !== undefined) {
+      const fips = breakdowns.filterFips as Fips;
+      if (fips.isState() && breakdowns.geography === "county") {
+        df = df.where(
+          (row) => row["county_fips"].substring(0, 2) === fips.code
```

Comment:
I'd prefer to keep the substring logic on the Fips class. One way to do that is to create a Fips instance to do the comparison, but in order to avoid creating so many objects I think a better way is to add a method `isParentOf(fipsCode: string)`

Reply:
o nice idea